internal/ext/unsafex/unsafex.go

```diff
@@ -17,42 +17,124 @@
 // Importing this package should be treated as equivalent to importing unsafe.
 package unsafex
 
-import "unsafe"
-
-// Size is like [unsafe.Sizeof], but it is a generic function and it returns
-// an int instead of a uintptr (Go does not have types so large they would
-// overflow an int).
-func Size[T any]() int {
```

Comment:
I'm confused by the change to remove this and add `Layout`. Nothing ever looks at `Layout.Align`, so is it really useful to always compute size and align together? They require separate calls to the underlying `unsafe` package, so seem totally fine to keep separate (especially since we actually only care about `Size` right now).

Reply:
Old habits die hard. I'll switch back to using just size.